utility/src/main/java/org/oppia/util/data/HtmlParser.kt

```diff
@@ -0,0 +1,34 @@
+package org.oppia.util.data
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.widget.TextView
+
+const val CUSTOM_TAG = "oppia-noninteractive-image"
+const val HTML_TAG = "img"
+const val CUSTOM_ATTRIBUTE = "filepath-with-value"
+const val HTML_ATTRIBUTE = "src"
+
+/** Html Parser for android TextView to parse Html tag. */
+class HtmlParser(
+  internal var context: Context
+) {
+
+  fun parseHtml(rawString: String?, tvContents: TextView): Spannable {
```

Comment:
Added documentation to this method, so that it becomes clear to other person what this function actually does, without looking at the internal code.

Reply:
added java doc comment.